### PR TITLE
evancookaudio sleepingdinosaur firmware v2

### PR DESCRIPTION
**Pull request by evanmcook**
## Description

revised firmware for evancookaudio sleepingdinosaur to match characteristics of v1 pcb, and to enable via in future. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/evancookaudio/sleepingdinosaur/sleepingdinosaur.h [hunk 1]
```diff
@@ -37,5 +37,5 @@
         {k06,   k07, k08, k09, k10,   k11},\
         {k12,   k13, k14, k15, k16,   k17},\
         {k18,   k19, k20, k21, k22,   k23},\
-        {KC_NO, k24, k25, k26, KC_NO, k27},\
+        {       k24, k25, k26,        k27},\
```
Comment (drashna) [inline keyboards/evancookaudio/sleepingdinosaur/sleepingdinosaur.h:40]: Please revert these changes.

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]
